pod/annotations.go

```diff
@@ -746,3 +746,17 @@ func IsScheduledInTrough(pod *corev1.Pod) bool {
 	_, ok := pod.Annotations[AnnotationKeyPodScheduledInTrough]
 	return ok
 }
+
+func RenderVersionInformation(kernelVersion string, executorVersion string) string {
```

Comment:
What codebase were you imagining calling this function?

Reply:
titus-executor has to do this, I can just move it to there.